graylog2-web-interface/src/components/search/MessageFieldDescription.jsx

```diff
@@ -54,6 +55,13 @@ const MessageFieldDescription = React.createClass({
 
     return termsMarkup;
   },
+  _getFieldValue() {
+    if (this.props.linkedFields.includes(this.props.fieldName)) {
+      return <a href={this.props.possiblyHighlight(this.props.fieldName)}>{this.props.possiblyHighlight(this.props.fieldName)}</a>
```

Comment:
You probably don't want to use the highlighted version of the field name as hyperlink reference. 😉

Reply:
Good catch. Will make that fix.